cfgov/templates/wagtailadmin/css/richtext.css

```diff
@@ -0,0 +1,3 @@
+.richtext {
+  max-width: 1000px
```

Comment:
No need to be sorry! I always forget the correct spacing, and luckily you always remind me! For future code audit purposes, I saw inconsistent inclusion/exclusion of the semi-colon in other short CSS files in this repo; they probably should be fixed.